reactive/kotlinx-coroutines-rx3/README.md

```diff
@@ -0,0 +1,91 @@
+# Module kotlinx-coroutines-rx3
+
+Utilities for [RxJava 3.x](https://github.com/ReactiveX/RxJava).
+
+Coroutine builders:
+
+| **Name**        | **Result**                              | **Scope**        | **Description**
+| --------------- | --------------------------------------- | ---------------- | ---------------
```

Comment:
Please fix the packages in this file in order to make knit task pass without warnings

Reply:
Done! I couldn't find which task in the knit task to run to test this locally though